src/theory/output_channel.h

```diff
@@ -146,10 +135,7 @@ class OutputChannel {
    *
    * @param n - a theory atom; must be of Boolean type
    */
```

Comment:
Maybe we should add a comment for some of the exceptions that the methods in `OutputChannel` can throw?

Reply:
This may end up being too generic to be informative. "An unrecoverable CVC4::Exception except where noted otherwise applies to almost all of CVC4." Did you have something else in mind?